providers/ms365/connection/sharepoint_report.go

```diff
@@ -17,20 +17,22 @@ var sharepointReport = `
 $ErrorActionPreference = "Stop"
 $token = '%s'
 $url = "%s"
-Install-Module PnP.PowerShell -Force -Scope CurrentUser -RequiredVersion 1.12.0
+Install-Module PnP.PowerShell -Force -Scope CurrentUser
 Import-Module PnP.PowerShell
 Connect-PnPOnline -AccessToken $token -Url $url
 
 $SPOTenant = (Get-PnPTenant)
 $SPOTenantSyncClientRestriction = (Get-PnPTenantSyncClientRestriction)
+$SPOSite = (Get-PnPTenantSite)
 
 $sharepoint = New-Object PSObject
 Add-Member -InputObject $sharepoint -MemberType NoteProperty -Name SPOTenant -Value $SPOTenant
 Add-Member -InputObject $sharepoint -MemberType NoteProperty -Name SPOTenantSyncClientRestriction -Value $SPOTenantSyncClientRestriction
+Add-Member -InputObject $sharepoint -MemberType NoteProperty -Name SPOSite -Value $SPOSite
```

Comment:
lets return all sites from the script. even it we dont end up using them its better to keep this script as simple as possible and do any other logic in the provider itself

Reply:
awesome idea, Thanks for taking care of it